src/GraphQLSchemaLinter.php

```diff
@@ -84,7 +84,7 @@ public function getVersion() {
         $future->setCWD($bin_root);
         list($stdout) = $future->resolvex();
         $json = json_decode($stdout, true);
-        $dep_version = $json['dependencies'][$name]['version'];
+        $dep_version = (isset($json['dependencies']) && isset($json['dependencies'][$name]) && isset($json['dependencies'][$name]['version'])) ? $json['dependencies'][$name]['version'] : null;
```

Comment:
I think you can use `idxv()` here:

```php
$dep_version = idxv($json, array('dependencies', $name, 'version');
```

Reply:
I was checking for the keys on other way, but yours is more idiomatic. Thanks.